dragonfly/item/register.go

```diff
@@ -83,4 +83,5 @@ func init() {
 	world.RegisterItem("minecraft:clay_ball", ClayBall{})
 
 	world.RegisterItem("minecraft:stick", Stick{})
-}
+	world.RegisterItem("minecraft:bricks", Brick{})
```

Comment:
I think this is supposed to be `minecraft:brick`

Reply:
Hm, it worked for me when I tested in the creative inventory.

Reply:
Because this is used on disk and not over network.